meteorite_fall.js

```diff
@@ -38,6 +42,13 @@ function loopGame() {
     }
     
     player.draw(context);
+
```

Comment:
Perhaps it would be better to put this into into a separate function called `pause()`. It would be nice to not fill loopGame() function with too much code - best to call methods inside of it.

Reply:
Look at the code though. Because the problem is I want to stop stuff from falling, and new meteorites from spawning. But I want them to still be drawn. I'll try and seperate but may have to change your algo code.